app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala

```diff
@@ -61,12 +64,13 @@ case class Registration(
       TaskStatus.CannotStartYet
 
   def numberOfCompletedSections: Int =
-    Array(isLiabilityDetailsComplete,
-          isCompanyDetailsComplete,
-          !isPartnershipWithPartnerCollection && isPrimaryContactDetailsComplete,
-          isGroup && isOtherOrganisationsInGroupComplete,
-          isPartnershipWithPartnerCollection && isNominatedPartnerDetailsComplete && isOtherPartnersDetailsComplete,
-          isRegistrationComplete
+    Array(
+      isLiabilityDetailsComplete,
+      isCompanyDetailsComplete,
+      !isPartnershipWithPartnerCollection && isPrimaryContactDetailsComplete,
+      isGroup && isOtherOrganisationsInGroupComplete,
+      isPartnershipWithPartnerCollection && isNominatedPartnerDetailsComplete && isOtherPartnersDetailsComplete,
+      isRegistrationComplete
     ).count(isComplete => isComplete)
```

Comment:
Looks we we do have some degree of is complete logic. (Just an observation).

Reply:
This doesn't appear to be working for members because with a partially complete member the section shows as completed. Another bug to resolve later (I have a list 🙂).